framework/src/main/java/org/checkerframework/framework/qual/HasQualifierParameter.java

```diff
@@ -1,11 +1,6 @@
 package org.checkerframework.framework.qual;
 
-import java.lang.annotation.Annotation;
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import java.lang.annotation.*;
```

Comment:
Our style is to not use wildcard imports.  Please just add one import statement.